engine/src/main/java/org/hibernate/validator/internal/xml/ClassLoadingHelper.java

```diff
@@ -49,11 +48,7 @@
 	private ClassLoadingHelper() {
```

Comment:
Wouldn't be nicer to make `ClassLoadingHelper` non static and pass an instance of it to the appropriate classes. Inside it can wrap the provided classloader and do the right thing.